tools/src/main/java/org/apache/kafka/tools/streams/StreamsGroupCommandOptions.java

```diff
@@ -33,13 +33,18 @@
 import static org.apache.kafka.tools.ToolsUtils.minus;
 
 public class StreamsGroupCommandOptions extends CommandDefaultOptions {
+    private static final String NL = System.lineSeparator();
     public static final Logger LOGGER = LoggerFactory.getLogger(StreamsGroupCommandOptions.class);
 
     public static final String BOOTSTRAP_SERVER_DOC = "REQUIRED: The server(s) to connect to.";
     public static final String GROUP_DOC = "The streams group we wish to act on.";
+    private static final String ALL_GROUPS_DOC = "Apply to all streams groups.";
+    private static final String INPUT_TOPIC_DOC = "The input topic whose streams group information should be deleted or topic that should be included in the reset offset process. " +
```

Comment:
```suggestion
    private static final String INPUT_TOPIC_DOC = "The input topic whose committed offset should be deleted or reset. In `reset-offsets` case, partitions can be specified using this format: `topic1:0,1,2`, where 0,1,2 are the partition to be included in the process. Multiple input topics can be specified. Supported operations: delete-offsets, reset-offsets."
```

Can you not use `delete-offsets` with specific partitions? Not sure what you meant by `The input topic whose streams group information should be deleted`.